schemas/file.yml

```diff
@@ -225,3 +225,17 @@
 
       short: A fork is additional data associated with a filesystem object.
       example: Zone.Identifer
+
+    - name: origin_referrer_url
+      level: extended
+      type: keyword
+      ignore_above: 8192
+      description: The URL of the webpage that linked to the file.
```

Comment:
I don't quite understand what does it mean? Could you elaborate it a bit, give some examples maybe? Is it url, where file was located before downloading?

Reply:
Hi @trisch-me,
Thank you for the comments/feedbacks!
Following is the actual example of the `file.origin_referrer_url` and `file.origin_url`.

When you download an image file (for example, `news30_img3.png`) from the following web page, 
* http://girls.seccon.jp/news30.html 

the  `file.origin_url` will be the url for the downloaded file, and the `file.origin_referrer_url` will be the url which hosted the file (http://girls.seccon.jp/news30.html).

<img width="727" alt="image" src="https://github.com/user-attachments/assets/cf3454e6-eb30-4576-9cab-5ea50e2470e6">

Reply:
thanks for explanation, can you add more information into description/examples of the actual fields?